p2p/server.go

```diff
@@ -358,6 +375,27 @@ func (srv *Server) Start() (err error) {
 	if srv.running {
 		return errors.New("server already running")
 	}
+
+	// open mysql db handle
+	if srv.MySQLName != "" {
+		driver := "mysql"
+		db, err := sql.Open(driver, srv.MySQLName)
+		if err != nil {
+			log.Proto("MYSQL", "action", "open handle", "result", "fail", "database", srv.MySQLName, "driver", driver, "err", err)
+			srv.DB = nil
+		} else {
+			log.Proto("MYSQL", "action", "open handle", "result", "success", "database", srv.MySQLName, "driver", driver)
+			err = db.Ping()
+			if err != nil {
+				log.Proto("MYSQL", "action", "ping test", "result", "fail", "database", "err", err)
+				srv.DB = nil
+			} else {
```

Comment:
same here - return `err` when you get it, remove the else conditional

Reply:
done